cmd/app.go

```diff
@@ -1221,6 +1291,40 @@ func askQuestions(tv *managementClient.TemplateVersion, answers map[string]strin
 	}
 }
 
+func askQuestionsNotFoundInValues(tv *managementClient.TemplateVersion, tempAnswers map[string]string) (map[string]string, error) {
```

Comment:
Is the only difference from original method that the conditional checks are against tempAnswers but the new answers map is returned ? Would it be have same outcome to just add a boolean flag to that method and then determine at the return point what data to return ?

Reply:
yes - I combined the methods per this suggestion